content/pt/docs/concepts/components.md

```diff
@@ -3,7 +3,6 @@ title: Componentes
 description: Os principais componentes que compõem o OpenTelemetry
 weight: 20
 default_lang_commit: f5c228e5d03deaabc00d5920c5757bf7bd23e3f3
```

Comment:
As I mentioned in https://github.com/open-telemetry/opentelemetry.io/pull/6514#pullrequestreview-2674463812:

> ... also update the `default_lang_commit` and then removing `drifted_from_default`?

`default_lang_commit` should be 99a39c5e4e51daba968bfbb3eb078be4a14ad363, unless I'm missing something.
Cf. https://github.com/open-telemetry/opentelemetry.io/commits/main/content/en/docs/concepts/components.md